### PR TITLE
Add zenodo metadata

### PR DESCRIPTION
**Pull request by saraedum**

### CODE REVIEW

**Comment by saraedum** [comment]
@linuxrider do you have an ORCID?

File: .zenodo.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+    "creators": [
+        {   
+            "orcid": "0000-0002-9686-3948",
+            "name": "Engstfeld, Albert Kilian",
+            "affiliation": "Universität Ulm"
```
Comment (DunklesArchipel) [inline .zenodo.json:6]: or Ulm University?

Comment (linuxrider) [reply .zenodo.json:6]: Ulm University

File: .zenodo.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+        },
+        {   
+            "orcid": "",
+            "name": "",
+            "affiliation": "Universität Ulm"
```
Comment (linuxrider) [inline .zenodo.json:11]:
```suggestion
            "orcid": "0000-0001-7119-1295 ",
            "name": "Johannes M. Hermann",
            "affiliation": "Ulm University"
```

File: .zenodo.json (hunk 1 above)
Comment (saraedum) [reply .zenodo.json:6]:
```suggestion
            "affiliation": "Ulm University"
```

**Review by DunklesArchipel — APPROVED** [review]
Looks good to me.

@linuxrider do you want your middle name spelled out?

**Comment by saraedum** [comment]
@linuxrider if you want this changed, feel free to open a PR.

**Comment by saraedum** [comment]
> Looks good to me.
> 
> @linuxrider do you want your middle name spelled out?

He proposed it like that, so I guess not.

Feel free to open a PR if you change your mind.

**Pull request merged**